docs/post-upgrade-req-cti-alerts

```diff
@@ -0,0 +1,40 @@
+[[post-upgrade-req-cti-alerts]]
+
+= Migrate detection alerts enriched with threat intelligence
+
+After upgrading to {stack} version 7.15.x from a previous minor release (e.g., 7.14.x, etc.), you need to migrate detection alerts enriched with threat intelligence to ensure threat intelligence properly displays in {elastic-sec}.
+
+To migrate detection alerts:
+
+. <<deactivate-detect-rules, Deactivate all detection rules>> (optional).
+
++
+
+IMPORTANT: All rules should be deactivated prior to upgrading your {stack}. Only complete this step if you haven't already.
```

Comment:
I'm a little confused: is it required or optional to deactivate all the detection rules before upgrading? Even if not strictly **required**, It sounds like this is a step that we really encourage customers to do, so maybe also calling it optional is misleading.

Reply:
Removing the note under step 1 and updating the step to:
Ensure that all detection rules are deactivated prior to upgrading your {stack}.